src/main/kotlin/com/openlattice/linking/controllers/RealtimeLinkingController.kt

```diff
@@ -52,6 +56,33 @@ class RealtimeLinkingController(
         return linkableEntitySets.minus(entitySetsNeedLinking)
     }
 
+    @RequestMapping(
+            path = [RealtimeLinkingApi.MISSING],
+            method = [RequestMethod.POST],
+            produces = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    override fun getEntitiesMissingLinking(@RequestBody entitySetIds: Set<UUID>): Map<UUID, Set<UUID>> {
+        ensureAdminAccess()
+        val linkableEntitySets = lqs
+                .getLinkableEntitySets(linkableTypes, entitySetBlacklist, whitelist.orElse(setOf()))
+                .toSet()
+        val linkableRequestedEntitySets = linkableEntitySets
+                .intersect(entitySetIds)
+        val nonLinkableRequestedEntitySets = entitySetIds.subtract(linkableEntitySets)
+        if(nonLinkableRequestedEntitySets.isNotEmpty()) {
+            logger.warn("The following entitySets were queried for missing entities, " +
```

Comment:
I would rephrase that to "for entities missing linking id" or "for entities not yet linked" missing entities is a bit misleading